generators/server/templates/src/test/resources/config/_application.yml

```diff
@@ -14,9 +14,14 @@
 eureka:
     client:
         enabled: false
+    instance:
```

Comment:
@PierreBesson this was not present in master but the travis builds are passing there but its failing for my PR thats why I have added this. any idea why